documentation/guide-validation.asciidoc

```diff
@@ -30,10 +32,23 @@ A description of how to enable BV can be found in the relevant http://docs.sprin
     </dependency>
 ----
 
-* Add the +@Validated+ annotation to the implementation (spring bean) to be validated. The standard use case is to annotate the logic layer implementation, i.e. the use case implementation or component facade in case of simple logic layer pattern. Thus, the validation will be executed for service requests as well as batch processing. For methods to validate go to their declaration and add constraint annotations to the method parameters.
+.*quarkus*
+[source,xml]
+----
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-hibernate-validator</artifactId>
+    </dependency>
+----
+
+* For methods to validate go to their declaration and add constraint annotations to the method parameters.
++
+__In spring applications you can add the `+@Validated+` annotation to the implementation (spring bean) to be validated (this is an annotation of the spring framework, so it`s not available in the Quarkus context). The standard use case is to annotate the logic layer implementation, i.e. the use case implementation or component facade in case of simple logic layer pattern. Thus, the validation will be executed for service requests as well as batch processing.__
```

Comment:
FYI: asciidoc had an old syntax for code by sourrounding with plus symbols (e.g. +@Validated+).
The new syntax for code that goes inline with text is to sourround with back-ticks (e.g. `@Validated`).
You should remove the plus signs after chaning to backticks. We still have leftovers in our older doc sections using the plus syntax that should be replaced with backticks. Also something worth to share with the team in the standup. Also it would be great if you could find some minutes to add this here: https://github.com/devonfw/docgen/blob/master/documentation/guidelines.asciidoc